src/biotite/structure/atoms.py

```diff
@@ -233,7 +233,7 @@ def _del_element(self, index):
         else:
             raise TypeError(f"Index must be integer, not '{type(index).__name__}'")
 
-    def equal_annotations(self, item):
+    def equal_annotations(self, item, equal_nan: bool = True):
```

Comment:
Biotite does not use type annotations (yet)

```suggestion
    def equal_annotations(self, item, equal_nan=True):
```

Reply:
Oops, thank you!